manuscript.Rmd

```diff
@@ -68,55 +68,154 @@ set.seed(42)
+Similar to peer-review for manuscripts, software code is amenable to peer-review.
+It is common practice in software development to have peers review code.
+For example, if a code base is changed, it is good practice to request a change with the developer team before actually changing the code base.
+The change is reviewed by experts and/or maintainers before it is integrated in the active branch of development.
+In developing these guidelines, we see the opportunity to foster such good practices and reward researchers that actively engage in reviewing computer code.
+Software peer-review is increasingly becoming recognized as a means to enable better research software.
+Peer-review can happen in the context of dedicated software journal (e.g, Journal of Statistical Software), as part of the standard review process (for guidelines, see https://www.carlboettiger.info/2013/06/13/what-I-look-for-in-software-papers.html), or as part of submissions to software archives that guarantee certain standards (e.g., rOpenSci).
+We suggest that active participation in the review of scientific software, independent of the involvement or maintenance of specific software products, should be a dimension of positive evaluation.
 
 ## Reproducibility
 
-->Aaron and Max
+# Measuring Impact
+In our opinion, total citation metrics, number of users, downloads per month and similar may provide a coarse measure for the impact of a software package, even though it is important to note the severe shortcoming we described above.
 
+Complementing the quantitative aspect of impact, we suggest to introduce a mandatory qualitative judgement on the impact of the software, which fits with what @gartner2022responsible proposed as indicator "merit / impact statement".
+In our view, researchers should be requested to indicate at least one (and up to three) empirical research projects that directly benefited from the software contributions they made.
```

Comment:
```suggestion
In our view, researchers should be requested to indicate at least one (and up to three)  research projects that directly benefited from the software contributions they made.
```

Is empierical central? E.g., SNLLS has benefitted lavaan, or a new method may be used in a simulation study.